python/ray/autoscaler/_private/util.py

```diff
@@ -27,16 +27,16 @@
 REQUIRED, OPTIONAL = True, False
 
 
-HEAD_TYPE_MAX_WORKERS_WARN_TEMPLATE = (
-    "Setting `max_workers` for node type"
-    " `{node_type}` to the global `max_workers` value of {max_workers}. To"
+HEAD_TYPE_MAX_WORKER_NODES_WARN_TEMPLATE = (
+    "Setting `max_worker_nodes` for node type"
+    " `{node_type}` to the global `max_worker_nodes` value of {max_worker_nodes}. To"
     " avoid spawning worker nodes of type `{node_type}`, explicitly set"
-    " `max_workers: 0` for `{node_type}`.\n"
-    "Note that `max_workers: 0` was the default value prior to Ray 1.3.0."
+    " `max_worker_nodes: 0` for `{node_type}`.\n"
+    "Note that `max_worker_nodes: 0` was the default value prior to Ray 1.3.0."
     " Your current version is Ray {version}.\n"
     "See the docs for more information:\n"
     "https://docs.ray.io/en/master/cluster/config.html"
```

Comment:
This link 404's for me.

Reply:
Thanks! Looks like this had been broken for a while.  Updated with some more relevant links